docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+# Web Monetization Extension Test Plan
+
+## Introduction
+
+The purpose of this document is to guide end-user testing of the Web Monetization browser extension.
+We focus on functional test cases. The goal is to report bugs, issues or differences observed from the expected behavior.
+
+## Audience
+
+Any person interested in testing the beta release of the Web Monetization extension.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to configure payments or disconnect from the wallet.
+3. When the extension facilitates paying a monetized web page, check how payments get distributed between the receiving wallets on the page.
+
+## Dependencies
+
+### Digital Wallets
+
+The main prerequisite for sending or receiving Web Monetization payments is to have a Web Monetization-enabled digital wallet. Web Monetization-enabled digital wallets are provided by licensed service providers. A wallet allows a user to send or receive Web Monetization payments. The wallet providers are regulated by the laws of the countries in which they operate.
+
+Below is a list of the available wallet providers if you want to use real money:
+
+- [Fynbos](https://wallet.fynbos.app/wallet)
+- [GateHub](https://gatehub.net/mobile)
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.  
+Learn more about the providers for [Web Monetization compatible wallets here](https://webmonetization.org/docs/resources/op-wallets/).
+
+#### Test Digital Wallet
+
+You have the option to use “play” money from the [Interledger Test Wallet application](https://rafiki.money/).  
+Here you can set up an account that is enabled for Web Monetization, and other Interledger functionality, without involving real money.
+
+You can learn more about the [Interledger Test Wallet here](https://rafiki.dev/integration/playground/testnet/).
+
+## Testing
+
+### Environment and Software
+
+The resources and tools that you will need to test Web Monetization depend on your desired role, and what you want to test.
+
+We’ve identified three roles you can play when using Web Monetization:
```

Comment:
Suggestion: If the only role we are wanting people to play is that of a website visitor, I suggest removing all the other information about website owner and developer. You also don't really need the table if website owner is the only thing listed in it.